File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.java
class Solution {
    public int longestConsecutive(int[] nums) {
        Set<Integer> set = new HashSet<>();
        for(int num: nums){
            set.add(num);
        }
        int longest = 0;
        for(int n: set){
            if(set.contains(n-1)){
                continue;
            }else{
                int counter = 1;
                while(set.contains(n+counter)){
                    counter++;
                }
                longest = Math.max(counter,longest);
            }
        }
        return longest;
    }
}

File: Data Structures & Algorithms/palindromic-substrings/submission-0.py
class Solution:
    def countSubstrings(self, s: str) -> int:
        counter = 0 
        dpArray = []
        for i in range(len(s)):
            dpArray.append([0]*len(s))
        for i in range(len(s)-1, -1, -1):
            for j in range(i, len(s)):
                if j-i == 0:
                    dpArray[i][j] = True
                    counter += 1
                elif j-i == 1:
                    if s[i] == s[j]:
                        dpArray[i][j] = True
                        counter += 1
                    else:
                        dpArray[i][j] = False
                else:
                    if s[i] == s[j] and dpArray[i+1][j-1]:
                        counter += 1
                        dpArray[i][j] = True
                    else:
                        dpArray[i][j] = False
        return counter
                    

